raster/r.neighbors/main.c

```diff
@@ -75,6 +81,15 @@ static struct menu menu[] = {
     {0, 0, 0, 0, 0, 0, 0, 0}
 };
 
+/* modify this table to add new methods */
+static struct weight_functions weight_functions[] = {
+    {"none", "No weighting"},
+    {"gaussian", "Gaussian weighting function"},
+    {"exponential", "Exponential weighting function"},
+    {"file", "File with a custom weighting matrix"},
```

Comment:
Function descriptions are not extracted for translation. This should be fixed.

Reply:
> Function descriptions are not extracted for translation. This should be fixed.

Thanks Maris for the feedback. I am no C dev. This has been mostly guided changes. I am grateful for any hints on how to make function descriptions translatable...

Reply:
I pushed the required change to your branch.